go.mod

```diff
@@ -516,6 +516,7 @@ replace (
 	github.com/Azure/azure-sdk-for-go/sdk/resourcemanager/consumption/armconsumption => github.com/elastic/azure-sdk-for-go/sdk/resourcemanager/consumption/armconsumption v1.1.0-elastic
 	github.com/apoydence/eachers => github.com/poy/eachers v0.0.0-20181020210610-23942921fe77 //indirect, see https://github.com/elastic/beats/pull/29780 for details.
 	github.com/dop251/goja => github.com/elastic/goja v0.0.0-20190128172624-dd2ac4456e20
+	github.com/elastic/go-quark => github.com/leehinman/go-quark v0.3.1
```

Comment:
This will have to be switched back to official elastic go-quark repo.  The only change between this and official v0.3.0 is a backport of https://github.com/elastic/go-quark/commit/12841a466bca204ef05ae117afefd18d9e26a761 which allows linking to the correct GLIBC version depending on OS arch.

Reply:
I wonder if we can just require the particular commit instead of a fork here, since we plan to go back to the elastic package anyway.